src/components/button/stories/button.README.md

```diff
@@ -48,5 +49,21 @@
 </Button>
 ```
 
+#### Button Shapes
+```jsx
```

Comment:
Thanks for documenting this, it'd be nice to also have a working demo in the `Button.stories.js` file, can you add them there as well?

Reply:
sure! Tomorow i'll do it